set_env.sh

```diff
@@ -0,0 +1,7 @@
+export SNORKELHOME="$HOME/Documents/snorkel"
```

Comment:
Okay let's see if we can remove the user-specific parts. If snorkel is a submodule, you can get its absolute path with

```sh
`pwd`/snorkel
```